ilastik/applets/dataSelection/dataSelectionSerializer.py

```diff
@@ -493,6 +521,16 @@ def unload(self):
             This way we can avoid invalid state due to a partially loaded project. """ 
         self.topLevelOperator.DatasetGroup.resize( 0 )
 
+    def _shouldRetrain(self):
```

Comment:
maybe add a beautiful '@property' here?

Reply:
this method should not be a part of  `dataSelectionSerializer` public API, so I'd rather keep it protected

Reply:
I don't see how '@property' would change anything about that. (It's always public anyway, the underscore can stay to indicate that it's meant to be private.)
Anyhow, it's fine as is 👍

Reply:
true! on the 2nd thought, I think the `@property` would be nice in here :)

Reply:
done :white_check_mark: